_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Creating dashboards
+nav_order: 30
+has_children: true
+---
+
+# Creating dashboards
+
+A dashboard combines multiple data visualizations into a single view. With the **Dashboard** application, you can:
```

Comment:
Technically, dashboards are not just limited to displaying visualizations (they can render almost anything in a panel), but this is fine as an approximation.

Reply:
Revised to reflect Dashboards broader capabilities.